### ISSUE TITLE
⌨ ROD final decicion not available with voice commands

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#14517]
### ⌨ Accessibility and Expected Behaviour

Task: Record the final decision of your candidate

**🛡️Fable tester reported:**
I cannot select the "qualify candidate" radio button by saying "click qualified candidate". I had to use the command "click radio button" to select the button. 

### WCAG 2.1 Criteria

2.5.3 Label in Name

### 📋 Steps to Reproduce

1. sign in 
2.Change your user profile from “Applicant” to “Community”
3. Navigate to “Processes”.
4. Search for and Select the process “Brooke’s Test”.
5. Navigate to “Screening and assessment”.
6. Select any applicant
7. Record the final decision of your candidate (Qualify the candidate)

### What operating systems are you encountering the problem on?

Windows

### What browsers are you encountering the problem on?

Chrome

### What assistive technologies are you encountering the problem on?

Dragon NaturallySpeaking

**Comment by gobyrne** [comment]
@bella77-69 are you able to reproduce this and propose a solution? Do we need to buy you Dragon naturally-speaking?

**Comment by bella77-69** [comment]
Hi @BrookeCameron after some investigation I think I know what is going on here (@esizer you can correct me if I am wrong 😉) 
When I use voice access and say "click qualify candidate", or "click radio button qualify candidate" I get numbered options instead of the radio button being selected. 
This is happening because voice access searches for text containing "qualify" or "candidate" and since both radio buttons match "candidate" voice access shows numbers for the users selection. .

**Comment by esizer** [comment]
> Hi [@BrookeCameron](https://github.com/BrookeCameron) after some investigation I think I know what is going on here ([@esizer](https://github.com/esizer) you can correct me if I am wrong 😉)
> When I use voice access and say "click qualify candidate", or "click radio button qualify candidate" I get numbered options instead of the radio button being selected.
> This is happening because voice access searches for text containing "qualify" or "candidate" and since both radio buttons match "candidate" voice access shows numbers for the users selection. .

I think you may be onto something with that @bella77-69 I think there could be two issues here related to what you have discovered:

 1. The unique words are pretty darn similar so voice access could be trying to make up for poor annunciation. It _thinks_ you may has missed the first syllable of "Dis-qualify" making it ambiguous
 2. Since the first word has some phoenetic overlap, it fallsback to the second word which _is_ ambigous

To make sure, can you try the voice access command "show names" and see if it does have the proper name?

This is a tough one! I'm not sure we have a great way out of this opther than changing copy to less ambigous terms phoentically but I doubt that is a great solution 😖
